src/main/java/uk/gov/justice/hmpps/casenotes/notes/CaseNoteAdminService.kt

```diff
@@ -22,28 +23,22 @@ class CaseNoteAdminService(
   private val eventPublisher: ApplicationEventPublisher,
 ) {
 
-  fun replaceCaseNote(personIdentifier: String, id: UUID, request: ReplaceNoteRequest): CaseNote {
-    val existing = noteRepository.findByIdAndPersonIdentifier(id, personIdentifier)
-      ?: throw EntityNotFoundException.withId(id.toString())
-
+  fun replaceCaseNote(id: UUID, request: ReplaceNoteRequest): CaseNote {
+    val existing = noteRepository.findByIdOrNull(id) ?: throw EntityNotFoundException.withId(id.toString())
     noteRepository.delete(existing)
     noteRepository.flush()
     return noteRepository.save(
       request.asNote(existing) { domain, code -> checkNotNull(subTypeRepository.findByKey(TypeKey(domain, code))) },
     ).also { eventPublisher.publishEvent(it.createEvent(UPDATED)) }.toModel()
   }
 
-  fun deleteNote(personIdentifier: String, caseNoteId: UUID) {
-    getCaseNote(personIdentifier, caseNoteId).also {
-      noteRepository.delete(it)
-      eventPublisher.publishEvent(it.createEvent(DELETED))
-    }
+  fun deleteNote(id: UUID) {
+    val existing = noteRepository.findByIdOrNull(id) ?: throw EntityNotFoundException.withId(id.toString())
```

Comment:
I saw you removed the 404 response from the controller but this line will still cause it. No real reason to check that it exists - we don't with sync